docs/csharp/language-reference/keywords/params.md

```diff
@@ -6,14 +6,16 @@ ms.date: 07/20/2015
 f1_keywords: 
   - "params_CSharpKeyword"
   - "params"
+  - "parameter array"
```

Comment:
I think this should probably be removed. It has to correlate to an actual keyword in the product.